src/templates/finops-hub/README.md

```diff
@@ -82,6 +84,63 @@ _<sup>1) A "scope" is an Azure construct that contains resources or enables purc
 
 <br>
 
+### Bicep Deploy with Private Endpoints
+
+#### Overview
+
+The purpose of this deployment is to deploy the FinOps Hub in an environment that does not allow public access.
+
+#### Prerequisites
+
+- The private deployment assumes that the target environment already has the virtual network where the private endpoints will be deployed.
+  - The virtual network needs to have two subnets; one for the private endpoints and one for the deployment scripts needed for the FinOps Hub.
+  - The virtual network for the deployment scripts needs to have the [needed configuration](https://learn.microsoft.com/azure/azure-resource-manager/bicep/deployment-script-vnet).
+    - A service endpoint configured for `Microsoft.Storage`
+    - A subnet delegation for `Microsoft.ContainerInstance/containerGroups`
```

Comment:
Could provide this as part of the deployment so customers can retro-fit to their environment rather than leaving it undefined.

Reply:
@MSBrett are you suggesting deploying the needed infra (vnet, dns zones ,....etc) ?

Reply:
I'd prefer to just ship it at a vnet which can be peered to a hub as required.  What's the minimum address space we need for this?

Reply:
I would like to have the option to use an existing VNet as well, because deploying a new VNet for a single purpose won't fit all environments, however, this will require some more work, but will give an option for all.

Reply:
fair enough